personal_project/_dbt_weather/profiles.yml

```diff
@@ -0,0 +1,13 @@
+weather_analysis:
+  outputs:
+    dev:
+      account: qzcrtyv-cr51063
```

Comment:
Account, user and password should be environment variables. Please find how to leverage them in dbt and change your password as these were exposed to the internet and we are not on a local container.